ballsdex/packages/balls/countryballs_paginator.py

```diff
@@ -38,7 +38,8 @@ def set_options(self, balls: List[BallInstance]):
             options.append(
                 discord.SelectOption(
                     label=f"{favorite}{shiny}{special}#{ball.pk:0X} {ball.countryball.country}",
-                    description=f"ATK: {ball.attack_bonus:+d}% • HP: {ball.health_bonus:+d}% • "
+                    description=f"ATK: {ball.attack}({ball.attack_bonus:+d}%) "
+                    f"• HP: {ball.health}({ball.health_bonus:+d}%) • "
                     f"Caught on {ball.catch_date.strftime('%d/%m/%y %H:%M')}",
```

Comment:
Just for some standardizing of code, precommit ideally should complain about this :p 

```suggestion
                    description=(
                        f"ATK: {ball.attack}({ball.attack_bonus:+d}%) "
                        f"• HP: {ball.health}({ball.health_bonus:+d}%) • "
                        f"Caught on {ball.catch_date.strftime('%d/%m/%y %H:%M')}"
                    ),
```